src/main/java/org/apache/commons/lang3/text/WordUtils.java

```diff
@@ -271,6 +274,11 @@ public static String wrap(final String str, final int wrapLength, final String n
      *               if blank string is provided a space character will be used
      * @return a line with newlines inserted, <code>null</code> if null input
      */
+    @SuppressWarnings({"assignment.type.incompatible","compound.assignment.type.incompatible"}) /*
+    #1 matcher is on the substring of str that starts with offset index, hence (offset + matcher.end())'s max value = offset + str.length() - 1 - offset = str.length() - 1
+    #2, #3 matcher is on the substring of str that starts with offset index, hence (offset + matcher.start())'s max value = offset + str.length() - 1 - offset = str.length() - 1
+    #4 matcher is on the substring of str that starts with offset + wrapLength index, hence (offset + wrapLength + matcher.start())'s max value = offset + wrapLength + str.length() - 1 - offset - wrapLength = str.length() - 1
+    */
```

Comment:
No, because the maximum value of spaceToWrapAt is str.length() - 1 and it does not depend on offset.

Reply:
I think it would be helpful to have that explicitly in this comment, because it wasn't obvious to me when reading the explanation, and makes it much more clear.